reconcile/aus/advanced_upgrade_service.py

```diff
@@ -116,15 +116,18 @@ def signal_reconcile_issues(
         dry_run: bool,
         org_upgrade_spec: OrganizationUpgradeSpec,
         exception: Exception,
-    ) -> None:
+    ) -> bool:
         """
         AUS will not fail on a reconcile issue. If issues should be noticed by an SRE team,
         alerts based on the metrics in the `reconcile.aus.metrics` module should be set up.
+
+        The function returns true to indicte that the exception was properly handled in the current context.
         """
         logging.error(
             f"Failed to reconcile cluster upgrades in OCM organization {org_upgrade_spec.org.org_id}",
             exc_info=exception,
         )
+        return True
```

Comment:
Do you have something planned? There is no False returned, why use a bool here?

Reply:
Ah, just saw `AdvancedUpgradeSchedulerBaseIntegration.signal_reconcile_issues`

Reply:
You add an attribute with a handler method, that is executed by `signal_reconcile_issues`. That method is registered in init. by that, you could avoid changing the implementation of `signal_reconcile_issues`.
Could make reading this code less of a surprise.
Example `signal_reconcile_issues`

```
if self.handler:
    return self.handler(dry_run, org_upgrade_spec, e)
return False
```

Reply:
i like the idea of extension points like handlers. in this case i would rather add some comments in the base function and overriden one to give the reader some hints about what is going on.